libmscore/input.cpp

```diff
@@ -116,7 +116,9 @@ void InputState::moveInputPos(Element* e)
             return;
 
       Segment* s;
-      if (e->isChordRest())
+      if (e->isSegment())
```

Comment:
The else is not useful anymore then.
Also, it's probably a good case to use the new toSegment(), toChord() everywhere in the function https://github.com/musescore/MuseScore/blob/master/mscore3.txt#L70